charts/seqr/templates/deployment.yaml

```diff
@@ -2,15 +2,9 @@
 apiVersion: apps/v1
 kind: Deployment
 metadata:
-  name: seqr
+  name: {{ include "seqr.fullname" . }}
   labels:
-    name: seqr
-    deployment: {{ .Release.Name }}
     {{- include "seqr.labels" . | nindent 4 }}
-  {{- if .Values.deploymentAnnotations }}
```

Comment:
doesn't exist anywhere.

Reply:
It's a fairly typical pattern in helm charts to provide empty annotations and tolerations, as these are things that end-deployers frequently need to change. Maybe it's better to provide an empty `{}` like the podAnnotations in the default values.yaml?

Reply:
👍 that's doable! 

I have this overwhelming question (kind of across this repo) about how we should go about supporting features that we ourselves don't use.  This is a good example where we haven't implemented `deploymentAnnotations` elsewhere (I guess by accident?) or tested it and it isn't provided by default by the `helm create`.   

I guess the best plan of action is to actually go through and make sure both `deploymentAnnotations` and `tolerations` are appropriately implemented on all charts/pods... which they definitely are not right now.

Reply:
Yeah, in general I feel like it's fine to draw the line somewhere... At a certain point, if people want to do stuff that's way outside what we anticipate, they should probably be building their own deployment.

But, I think a good minimum is to ensure that people can pass in arbitrary labels, annotations, and tolerations on most of the chart's resources. I think a lot of people will end up needing to provide those at some point for one reason or another, and having the settings will keep them from needing to fork the chart. So, I guess a good barometer here is asking "_should_ someone have to fork the chart to do this?"

- e.g. labels, annotations, adding name suffixes to things, probably not?
- e.g. "I need to make seqr use MongoDB for [webscale](https://www.youtube.com/watch?v=b2F-DItXtZs)", probably time for your own chart

Reply:
+1 this makes sense.  

My gut on this is that we want to make this as simple and dumb as possible to try to emulate a one-click install for people who don't know anything about helm or kubernetes.  I'd love if we were able to support the "I need this pod on this machine type" with just `nodeSelector` and `affinity`, leaving out `tolerations`, but I can see the use for supporting it.  Hopefully with the better unit testing we can at least get some coverage over those features so they're not just in the way.